src/main/java/com/work/management/entity/Employee.java

```diff
@@ -0,0 +1,53 @@
+package com.work.management.entity;
+
+import lombok.*;
+
+import javax.persistence.*;
+import java.util.Date;
+
+
+@Entity
+@Table(name = "EMPLOYEES")
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@EqualsAndHashCode
+public final class Employee {
```

Comment:
if you have..@Data..you don't need @equals an haschode..just check once..and remove it.